src/main/java/com/softserve/bookstore/controllers/BookController.java

```diff
@@ -54,11 +53,11 @@ public ResponseEntity<String> findAuthorId() throws SQLException, IOException {
         }
     }
 
-    @ExceptionHandler({BookNotFoundException.class})
-    public ResponseEntity<ErrorResponse> handleBookEx(BookNotFoundException msg) {
-        Logger.error(" Failed to find the Book ", msg);
-        return new ResponseEntity<>(new ErrorResponse(HttpStatus.NOT_FOUND, msg.getMessage()), HttpStatus.NOT_FOUND);
-    }
+//    @ExceptionHandler({BookNotFoundException.class})
+//    public ResponseEntity<ErrorResponse> handleBookEx(BookNotFoundException msg) {
+//        Logger.error(" Failed to find the Book ", msg);
+//        return new ResponseEntity<>(new ErrorResponse(HttpStatus.NOT_FOUND, msg.getMessage()), HttpStatus.NOT_FOUND);
+//    }
 
     @ExceptionHandler({SQLException.class})
     public ResponseEntity<ErrorResponse> handleSQlEx(SQLException msg) {
```

Comment:
also abbreviation thing
it is alright to use common abbreviations like Http or SQL, but try to stick to camel case

in this case handleSqlException is fine. and also use full name for exception, not ex